frontend/src/pages/Overview/TableView.js

```diff
@@ -73,81 +77,123 @@ const ProjectButtons = ({ project, showEditDialog, showProjectPermissions, allow
   );
 };
 
-const columns = [
-  // Documentation: https://react-data-table-component.netlify.app/?path=/docs/api-columns--page
-  {
-    name: (
-      <Typography variant="subtitle2" sx={{ fontSize: "1.1rem" }}>
-        {strings.common.project}
-      </Typography>
-    ),
-    selector: row => row.data.projectName,
-    sortable: true,
-    compact: false,
-    minWidth: "15rem",
-    cell: row => <Typography data-test="project-name">{row.data.projectName}</Typography>
-  },
-  {
-    name: (
-      <Typography variant="subtitle2" sx={{ fontSize: "1.1rem" }}>
-        {strings.common.status}
-      </Typography>
-    ),
-    selector: row => row.data.projectStatus,
-    sortable: true,
-    compact: true,
-    minWidth: "5rem",
-    cell: row => <Typography>{row.data.projectStatus}</Typography>
-  },
-  {
-    name: (
-      <Typography variant="subtitle2" sx={{ fontSize: "1.1rem" }}>
-        {strings.common.created}
-      </Typography>
-    ),
-    selector: row => row.data.creationUnixTs, // time in ms to use the built-in sort
-    sortable: true,
-    compact: true,
-    minWidth: "10rem",
-    cell: row => <Typography>{row.data.createdDate}</Typography> // formatted date that is shown
-  },
-  {
-    name: (
-      <Typography variant="subtitle2" sx={{ fontSize: "1.1rem" }}>
-        {strings.common.assignee}
-      </Typography>
-    ),
-    selector: row => row.data.assignee,
-    sortable: true,
-    compact: true,
-    minWidth: "5rem",
-    cell: row => <Typography>{row.data.assignee}</Typography>
-  },
-  {
-    name: (
-      <Typography variant="subtitle2" sx={{ fontSize: "1.1rem" }}>
-        {strings.common.tags}
-      </Typography>
-    ),
-    sortable: false,
-    compact: true,
-    minWidth: "0rem",
-    maxWidth: "20rem",
-    cell: row => row.components.Tags
-  },
-  {
-    name: (
-      <Typography variant="subtitle2" sx={{ fontSize: "1.1rem" }}>
-        {strings.common.actions}
-      </Typography>
-    ),
-    sortable: false,
-    right: true,
-    compact: false,
-    minWidth: "10rem",
-    cell: row => row.components.ProjectButtons
-  }
-];
+const TableViewEditor = ({ showTags, setShowTags, showBudgets, setShowBudgets }) => {
+  return (
+    <Box>
+      <FormGroup>
+        <FormControlLabel
+          control={<Checkbox checked={showTags} onChange={e => setShowTags(e.target.checked)} />}
+          label="Tags"
+        />
+        <FormControlLabel
+          control={<Checkbox checked={showBudgets} onChange={e => setShowBudgets(e.target.checked)} />}
+          label="Budgets"
+        />
+      </FormGroup>
+    </Box>
+  );
+};
+
+const useRawColumns = () => {
+  const rawColumns = [
+    // Documentation: https://react-data-table-component.netlify.app/?path=/docs/api-columns--page
+    {
```

Comment:
rawColumns must be returned by a hook, otherwise the string object will not be up2date in this object if the language has been changed